enderio-base/src/main/java/com/enderio/base/common/tag/EIOTags.java

```diff
@@ -1,3 +1,33 @@
 package com.enderio.base.common.tag;
 
-public class EIOTags {}
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.tags.ItemTags;
+import net.minecraft.world.item.Item;
+import net.minecraftforge.common.Tags.IOptionalNamedTag;
+
+public class EIOTags {
+    
+    public static void init(){
+        Items.init();
+        Blocks.init();
+    }
+    
+    public static class Items{
+        
+        private static void init(){}
+        
+        public static final IOptionalNamedTag<Item> DUSTS_LAPIS = ItemTags.createOptional(new ResourceLocation("forge", "dusts/lapis"));
+        public static final IOptionalNamedTag<Item> DUSTS_COAL = ItemTags.createOptional(new ResourceLocation("forge", "dusts/coal"));
+        public static final IOptionalNamedTag<Item> DUSTS_QUARTZ = ItemTags.createOptional(new ResourceLocation("forge", "dusts/quartz"));
+        public static final IOptionalNamedTag<Item> SILICON = ItemTags.createOptional(new ResourceLocation("forge", "silicon"));
+        public static final IOptionalNamedTag<Item> FUSED_QUARTZ = ItemTags.createOptional(new ResourceLocation("forge", "fused_quartz"));
```

Comment:
Should fused_quartz and clear_glass be in the enderio namespace? I don't see other mods adding glass types.